backend/src/app.module.ts

```diff
@@ -43,6 +44,7 @@ const FRONTEND_PATH = join(__dirname, "..", "..", "frontend", "build");
     }),
     RedisServiceModule,
     PrismaServiceModule,
+    JudgeServiceModule,
```

Comment:
Shouldn't need to import service module at the root module unless is a global connection.
```suggestion
```